extension/src/setup.ts

```diff
@@ -209,38 +201,149 @@ const warnUserCLIInaccessible = async (
   }
 }
 
-const extensionCanRunPythonCli = async (extension: IExtension, cwd: string) => {
-  let canRunCli = false
-  if (await extension.isDvcPythonModule()) {
-    try {
-      canRunCli = await extension.canRunCli(cwd)
-    } catch {}
+const warnUserCLIInaccessibleAnywhere = async (
+  extension: IExtension,
+  globalDvcVersion: string | undefined
+): Promise<void> => {
+  const binPath = await getPythonBinPath()
+
+  return warnUserCLIInaccessible(
+    extension,
+    true,
+    `The extension is unable to access an appropriate version of the CLI. The CLI was not located using the interpreter provided by the Python extension. ${
+      globalDvcVersion ? globalDvcVersion + ' is' : 'The CLI is also not'
+    } installed globally. For auto Python environment activation ensure the correct interpreter is set. Active Python interpreter: ${binPath}.`
```

Comment:
Maybe I'm wrong, but should there be a comma in this sentence?

```suggestion
    } installed globally. For auto Python environment activation, ensure the correct interpreter is set. Active Python interpreter: ${binPath}.`
```

This message is also rather lengthy 🤔 Should we try to simplify the message? First idea that   comes to mind is combining the 2nd and 3rd sentences 🤷

Reply:
I have dropped a few words but I am concerned that if we change the structure of the message that it will be less clear. This message/edge case is the whole reason this PR was created. It should not be seen often but it should get users out of trouble.